### ISSUE TITLE
Add additional Button to the UI

### ISSUE THREAD

**Issue by Icebird2000** [bryceco/GoMap#651]
Is there a way to add a button to the UI to change between “Editor with Aerial” and “Aerial” with only and vice versa in one step? Possibly customizable for other functions.

**Comment by bryceco** [comment]
You can do long-press on the Display button but it is still a two-tap sequence.

**Comment by Icebird2000** [comment]
Yes, i know this one, but it is slower than pressing the Display button and choose the Options. 
That’s why i ask for a fast switch between the display options. I think it’s helpful for everybody.

**Comment by bryceco** [comment]
I really don't want to add a new button to the main screen, so you'll have to come up with a different way to do this. Maybe long-press on the Bing (or whatever) icon? Some other gesture? I'm open to ideas.

Don't say to add it as a user preference: then it it becomes two new buttons.

**Comment by Icebird2000** [comment]
Ok. This is understandable.
I‘ve collected some possible Options for this:
- double tap on the screen with two fingers
- double tap on the screen with tree fingers
- press the map button and swipe up for map only
- press the map button and swipe down for map with editor
- double tap with one finger on the map button 
- long tap on the compass north button to switch between Map with editor and map only without asking

**Comment by patricks** [comment]
It would be also interesting why this background map type switch needs to be such a super quick action?

**Comment by R3gi** [comment]
How about merging the Display and Settings buttons so there'll be space for the desired button while keeping the same amount of buttons?

IMO the content of the Display screen can be still considered as settings and personally I don't see a reason why it's split anyways…

**Comment by ooosssay** [comment]
Why not when holding down the display button it should rotate between a favorited choice with customizable buttons like, for example, switching between Esri World Imagery Aerial only to Bing Aerial with editor. I would like this as in my area Bing is really high quality but dated while Esri World Imagery is more recent and the imagery was captured during winter making it easy to map with no trees blocking my view but the view is blurry and somewhat low quality.
